File: front/webav360/src/app/Models/AvaliacaoPublica.ts
import { Grupo } from "./Grupo";
import { Criterio } from "./Criterio";
import { Turma } from "./Turma";

export interface AvaliacaoPublica {
  sessaoId: number,
  turmaId: number,
  turma: Turma;
  grupos: Grupo[],
  criterios: Criterio[]
}

File: back/Application/DTOs/AvaliacaoPublica.cs
namespace Application.DTOs;

public class AvaliacaoPublicaDTO
{
    public int SessaoId { get; set; }
    public int TurmaId { get; set; }
    public TurmaDTO? Turma { get; set; }
    public IEnumerable<GrupoDTO> Grupos { get; set; } = [];
    public IEnumerable<CriterioDTO> Criterios { get; set; } = [];
}